src/main/java/cubicchunks/CommonEventHandler.java

```diff
@@ -45,7 +45,7 @@ public void onWorldLoad(WorldEvent.Load evt) {
 		ICubicWorld world = (ICubicWorld) evt.getWorld();
 
 		WorldType type = evt.getWorld().getWorldType();
-		if(type instanceof ICubicWorldType){
+		if(type instanceof ICubicWorldType) {
```

Comment:
No matter how it's configured, eclipse formatter will change things that IDEA doesn't change. I can try to make the same settings for eclipse and IDEA later and then reformat the code but it's going to be hard for now.